### DISCUSSION TITLE
How to add description to the parameters of slash commands in nextcord?

### DISCUSSION THREAD

**Discussion by FiziQaiser** [nextcord/nextcord#1072] [Q&A]
How to add description to the parameters of slash commands in nextcord?

**Comment by DenverCoder1** [comment]
There are multiple ways.

One way is by using SlashOption with the description parameter. Another is by documenting the parameters using Google, Numpy, or Sphinx-style docstrings.

I recommend the second option - see more on this feature: https://github.com/nextcord/nextcord/pull/480

https://docs.nextcord.dev/en/stable/interactions.html
